src/zc/buildout/buildout.py

```diff
@@ -1106,7 +1234,16 @@ def setup(self, args):
     runsetup = setup # backward compat.
 
     def annotate(self, args=None):
-        _print_annotate(self._annotated)
+        if args and "--verbose" in args:
+            verbose = True
+        else:
+            verbose = False
+        section = None
+        if args:
+            for arg in args:
+                if arg.startswith('--section'):
+                    _, section = arg.split("=")
```

Comment:
I don't see mention of '=' in reference.

if the option is repeated, the last one wins? :(

I don't think flags are needed. Buildout's verbosity (aka log level) should be used.

Why not just accept a list of sections?  If TCBOO, then error if more than one is provided.